integration/airflow/openlineage/airflow/utils.py

```diff
@@ -118,7 +120,7 @@ def execute_git(cwd, params):
     return out.decode('utf8').strip()
 
 
-def get_connection_uri(conn: Connection):
```

Comment:
Do we not want to keep the type of method params? This is more of a general comment, as I noticed we drop the type in other methods as well in the PR.

Reply:
From typing point of view, we would want to. Hovewer, we need to import them for this. Due to the fact that Python forbids circular imports, and the fact that initialization of our classes in Airflow integration is different between DAG integration and LineageBackend integration (and even worse in upcoming combination of both approaches) I'd refrain from using importing anything from Airflow as much as possible, and when impossible, do a local import.